openapi.yaml

```diff
@@ -2005,6 +1883,312 @@ paths:
+
+        This command can only be accessed by the unrestricted users of an account.
+
+        There are several conditions that must be met in order to successfully create a transfer request:
+
+        1. The account creating the transfer must not have a past due balance or active Terms of Service violation.
+
+        1. The service must be owned by the account that is creating the transfer.
+
+        1. The service must not be assigned to another Service Transfer that is pending or that has been accepted and is
+        incomplete.
+
+        1. Linodes must not:
+
+            * be assigned to a NodeBalancer, Firewall, or Managed Service.
+
+            * have any attached Block Storage Volumes.
+
+            * have any shared IP addresses.
+
+            * have any assigned /56, /64, or /116 IPv6 ranges.
+      operationId: createServiceTransfer
+      x-linode-cli-action: create
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - account:read_write
+      requestBody:
+        description: The services to include in this transfer request.
+        content:
+          application/json:
+            schema:
+              required:
+              - entities
+              type: object
+              properties:
+                entities:
+                  $ref: '#/components/schemas/ServiceTransfer/properties/entities'
+      responses:
+        '200':
+          description: >
+            Returns a Service Transfer object for the request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceTransfer'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Content-Type: application/json" \
+            -H "Authorization: Bearer $TOKEN" \
+            -X POST -d '{
+              "entities": {
+                "linodes": [
+                  111,
+                  222
+                ]
+              }
+            }' \
+            https://api.linode.com/v4/account/service-transfers
+      - lang: CLI
+        source: >
+          linode-cli service-transfers \
+            create \
+            --entities.linodes 111 \
+            --entities.linodes 222
+  /account/service-transfers/{token}:
+    x-linode-cli-command: service-transfers
+    parameters:
+      - name: token
+        in: path
+        description: The UUID of the Service Transfer.
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      x-linode-grant: unrestricted only
+      tags:
+      - Account
+      summary: Service Transfer View
+      description: |
+        Returns the details of the Service Transfer for the provided token.
+
+        This command can only be accessed by the unrestricted users of the accounts that created or accepted this
```

Comment:
This needs a bit more clarification. If the transfer is pending, any unrestricted user may view it. This is because we don't know who the transfer is intended for. In any other state, only the creator/acceptor accounts' unrestricted users may view it.

Reply:
Is this clearer?
"Until a transfer is accepted, unrestricted users of the account that created it can view it. After a transfer has been accepted, it can also be viewed by any unrestricted user of the account that accepted the transfer."

Reply:
@bbiggerr Carissa means any unrestricted user at all. Meaning any user for any customer.

Reply:
I think this is the clearest way to put it:

While a transfer is pending, any unrestricted user *of any account* can access this command. After a
transfer has been accepted, it can only be viewed by unrestricted users of the accounts that created and
accepted the transfer. If cancelled or expired, only unrestricted users of the account that created the
transfer can view it.